tools/gn

```diff
@@ -335,19 +339,21 @@ def main(argv):
     '--check',
   ]
 
-  if sys.platform == 'darwin':
-    # On the Mac, also generate Xcode projects for ease of editing.
+  if args.ide != '':
+    command.append('--ide=%s' % args.ide)
+  elif sys.platform == 'darwin':
+    # On the Mac, generate an Xcode project by default.
     command.append('--ide=xcode')
-
-  if sys.platform.startswith('win'):
-    # On Windows, also generate Visual Studio project for ease of editing.
+  elif sys.platform.startswith('win'):
+    # On Windows, generate a Visual Studio project.
     command.append('--ide=vs')
 
   gn_args = to_command_line(to_gn_args(args))
   out_dir = get_out_dir(args)
-  print "gn gen --check in %s" % out_dir
   command.append(out_dir)
   command.append('--args=%s' % ' '.join(gn_args))
+  print "Generating GN files in: %s" % out_dir
```

Comment:
Can you put these outputs behind a verbose flag?

Reply:
Spoke to Jason and we agreed that we should just print the out directory. If folks need to see the GN gen arguments, they can view it in the arg.gn file.